test/src/org/labkey/test/pages/cds/XAxisVariableSelector.java

```diff
@@ -126,9 +130,88 @@ public void setCellType(String... value)
         super.setAssayDimension(XPATHID, AssayDimensions.CellType, value);
     }
 
-    public void setAxisType(String... value)
+    public void setPlotType(String... value)
     {
-        super.setAssayDimension(XPATHID, AssayDimensions.AxisType, value);
+        super.setAssayDimension(XPATHID, AssayDimensions.PlotType, value);
+    }
+
+    public List<String> getTimePointPlotTypeOptions()
+    {
+        return getTimePointAdvancedOptions("Plot type");
+    }
+
+    public String getTimePointAdvancedOptionSelectedValue(String optionLabel)
+    {
+        _test.mouseOver(Locator.tagWithText("div", optionLabel + ":"));
+        String xpathDimField = getTimeOptionAdvancedOptionXPathDimField(optionLabel);
+        Locator.XPathLocator locDimField = Locator.xpath(xpathDimField);
+        if (_test.isElementPresent(locDimField))
+        {
+            WebElement element = locDimField.findElement(_test.getDriver());
+            if (element != null)
+                return element.getText();
+        }
```

Comment:
Use `locDimField.findElementOrNull` to find elements that may not be present to avoid finding an element multiple times; In this case, when the element is present, the test will find the element once during `isElementPresent` then again when you call `findElement` to actually get the WebElement.